src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -160,6 +160,9 @@
 
     @Override
     public void visitToken(DetailAST ast) {
+        if (ast != null) {
```

Comment:
Don't know whats going on, but it should be impossible for a `visitToken` to give a `null` AST.